src/event/event_config.h

```diff
@@ -133,7 +133,7 @@
 #	define EV_ONESHOT				0x0010
 #	define EV_CLEAR					0x0020
 #	define EV_DISPATCH				0x0080
-#	define EV_UDATA_SPECIFIC		0x0100
+#	define EV_UDATA_SPECIFIC			0x0000
```

Comment:
(and in particular doing it that way will also cause EV_VANISHED to be 0x0000 on Linux. currently it is 0x0200 via the #define at the top of the file)